driver/test/java/io/crate/client/jdbc/integrationtests/CrateJDBCConnectionTest.java

```diff
@@ -0,0 +1,589 @@
+            "not in ('pg_catalog', 'sys', 'information_schema', 'blob')");
+        while (rs.next()) {
+            connection.createStatement().execute(String.format("drop table if exists \"%s\".\"%s\"",
+                rs.getString("schema_name"), rs.getString("table_name")));
+        }
+    }
+
+    public static void setUpTables() throws InterruptedException, SQLException {
+        connection.createStatement().execute("create table if not exists test (" +
+            " id integer primary key," +
+            " string_field string," +
+            " boolean_field boolean," +
+            " byte_field byte," +
+            " short_field short," +
+            " integer_field integer," +
+            " long_field long," +
+            " float_field float," +
+            " double_field double," +
+            " timestamp_field timestamp," +
+            " object_field object as (\"inner\" string)," +
+            " ip_field ip" +
+            ") clustered by (id) into 1 shards with (number_of_replicas=0)");
+        waitForShards();
+
+        connection.createStatement().execute("create table if not exists arrayTest (" +
+                " id integer primary key," +
+                " str_array array(string)," +
+                " bool_array array(boolean)," +
+                " byte_array array(byte)," +
+                " short_array array(short)," +
+                " integer_array array(integer)," +
+                " long_array array(long)," +
+                " float_array array(float)," +
+                " double_array array(double)," +
+                " timestamp_array array(timestamp)," +
+                " obj_array array(object)" +
+                ") clustered by (id) into 1 shards with (number_of_replicas=0)");
+        waitForShards();
+    }
+
+    private static void waitForShards() throws InterruptedException, SQLException {
+        while (countUnassigned() > 0) {
+            Thread.sleep(100);
+        }
+    }
+
+    private static Long countUnassigned() throws SQLException {
+        ResultSet rs = connection.createStatement()
+            .executeQuery("SELECT count(*) FROM sys.shards WHERE state != 'STARTED'");
+        rs.next();
+        return rs.getLong(1);
+    }
+
+    @Before
+    public void before() throws SQLException {
+        insertIntoTable();
+    }
+
+    @After
+    public void after() throws SQLException {
+        deleteFromTable();
+    }
+
+    private static void insertIntoTable() throws SQLException {
+        Map<String, Object> objectField = new HashMap<String, Object>() {{
+            put("inner", "Zoon");
+        }};
+        PreparedStatement preparedStatement =
+            connection.prepareStatement("insert into test (id, string_field, boolean_field, byte_field, " +
+                "short_field, integer_field, long_field, float_field, double_field, object_field, " +
+                "timestamp_field, ip_field) values " +
+                "(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)");
```

Comment:
looks like that the complexity of this table (i mean all the columns and types) is only used at 1 test in that class.
would prefer a much simpler generic table used by most tests, and a complex one just created where it is needed (maybe same for the array table below)